site/content/docs/concepts/pipelines.md

```diff
@@ -115,6 +112,8 @@ This parses your `pipeline.yml`, creates a CodePipeline in the same account and
 
 ![Your completed CodePipeline](https://user-images.githubusercontent.com/828419/71861318-c7083980-30aa-11ea-80bb-4bea25bf5d04.png)
 
+Note: If you have selected a GitHub or Bitbucket repository, Copilot will help you connect to your source code with [CodeStar Connections](https://docs.aws.amazon.com/dtconsole/latest/userguide/welcome-connections.html). You will need to install the AWS authentication app on your third-party account and update the connection status. Copilot and the AWS Management Console will guide you through these steps.
```

Comment:
instead of "Note:" should we use [!!!note](https://squidfunk.github.io/mkdocs-material/reference/admonitions/)

```
!!!note
   If you have selected a GitHub or Bitbucket repository,...
```

Reply:
Fun! Maybe I'll use `!!!info` instead? That's what we use elsewhere, and it is giving more info.